vcbuild.bat

```diff
@@ -75,13 +71,24 @@ shift
 goto next-arg
 
 :args-done
-if defined upload goto upload
 
-if "%config%"=="Debug" set debug_arg=--debug
-if "%target_arch%"=="x64" set msiplatform=x64
-if defined nosnapshot set nosnapshot_arg=--without-snapshot
-if defined noetw set noetw_arg=--without-etw& set noetw_msi_arg=/p:NoETW=1
-if defined noperfctr set noperfctr_arg=--without-perfctr& set noperfctr_msi_arg=/p:NoPerfCtr=1
+if defined build_release (
```

Comment:
Unlike v0.12, we don't  set `build_release` when `build-release` is passed as argument, so this if statement will never be true. See `build-release` above. Best to check this after v0.10-staging is rebased on v0.10 though.